doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -307,8 +309,58 @@ Most cloud storage architectures incorporate a number of clustered storage nodes
 
 <a name="3.5"></a>
 ## 3.5. Cloud Topology
-Content to be developed along the following lines
-*	How do we define the different deployment types?
-    - Edge
-    -	Core DC 
-    -	etc.
+
+Please note: This section focuses on central DC based cloud deployments, Edge is currently out of scope.
+
+In order to provide the expected availability for any given service, a number of different OpenStack deployment topologies can be considered.  This section explores the main options and highlights the characteristics of each.  Ultimately the decision rests with the operator to achieve specific availability target taking into account use case, data centre capabilities, economics and risks.
+
+Availability of any single OpenStack cloud is dependent on a number of factors including: 
+-	environmental – dual connected power and PDUs, redundant cooling, rack distribution etc. 
+-	resilient network fabric – ToR (leaf), spine, overlay networking, underlay networking etc.   It is assumed that all network components are designed to be fault tolerant and all OpenStack controllers, computes and storage are dual-homed to alternate leaf switches.
+-	controller nodes setup in-line with the vendor recommendation (e.g. min 3 physical nodes)
+-	network nodes (where applicable) 
```

Comment:
@iangardner22 
- Controller nodes setup should be HA based on quorum ( aligned with vendor implementation )